docs/docs/50-roadmap.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Roadmap
+Description: Kargo Roadmap
+---
+
+# Roadmap
+
+Kargo does not currently have a regular release cadance. The plan is to do a few releases, and based on the resluts, create a more formal release cadance.
+
+:::caution
+This roadmap is subject to change at any time, for the most up to date information, please see the [GitHub Project](<link goes here>)
```

Comment:
@kencochrane Will put the link to the milestones here.

Reply:
https://github.com/akuity/kargo/milestones